tests/gnrc_dhcpv6_client_6lbr/tests/01-run.py

```diff
@@ -123,6 +126,13 @@ def testfunc(child):
     # and it is asking for a prefix delegation
     assert DHCP6OptIA_PD in pkt
 
+    # MUD URL option in DHCPv6 is not yet supported by scapy
+    assert DHCP6OptUnknown in pkt
+    mud_packet = pkt[DHCP6OptUnknown]
```

Comment:
```suggestion
    assert DHCP6OptMUD in pkt
    mud_packet = pkt[DHCP6OptMUD]
```

Reply:
This is a very cool idea :) I added it in the latest fixup commit